### PR TITLE
Add functionality to get /slabs

### PR DESCRIPTION
**Pull request by faisal-memon**
### Proposed changes
Adding support for /slabs API endpoint so we can monitor shared memory zone usage.

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/nginx-plus-go-client/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that all unit tests pass after adding my changes
- [x] I have updated necessary documentation
- [x] I have rebased my branch onto master
- [x] I will ensure my PR is targeting the master branch and pulling from my branch from my own fork

### CODE REVIEW

File: client/nginx.go [hunk 1]
```diff
@@ -910,6 +925,11 @@ func (client *NginxClient) GetStats() (*Stats, error) {
 		return nil, fmt.Errorf("failed to get stats %v", err)
 	}
 
+	slabs, err := client.getSlabs()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get stats %v", err)
```
Comment (Rulox) [inline client/nginx.go]:
```suggestion
		return nil, fmt.Errorf("failed to get stats: %v", err)
```

Just for consistency with the rest

**Comment by Rulox** [comment]
lgtm, a small suggestion on a error string and :shipit: !

File: client/nginx.go [hunk 2]
```diff
@@ -140,6 +141,20 @@ type Connections struct {
 	Idle     uint64
 }
 
+// Slabs is map of slab stats by zone name.
+type Slabs map[string]Slab
+
+// Slab represents slab related stats.
+type Slab struct {
```
Comment (pleshakov) [inline client/nginx.go]: @faisal-memon 
was it intentional to not include slots stats?

Comment (faisal-memon) [reply client/nginx.go]: good catch. i'm not using slots stats but Ill add that in.

File: client/nginx.go (hunk 1 above)
Comment (faisal-memon) [reply client/nginx.go]: done

File: client/nginx.go (hunk 2 above)
Comment (faisal-memon) [reply client/nginx.go]: done

**Pull request merged**